jbmc/src/java_bytecode/java_static_initializers.cpp

```diff
@@ -10,11 +10,12 @@ Author: Chris Smowton, chris.smowton@diffblue.com
 #include "java_object_factory.h"
 #include "java_utils.h"
 #include <goto-programs/class_hierarchy.h>
-#include <util/std_types.h>
-#include <util/std_expr.h>
+#include <json/json_parser.h>
```

Comment:
I'd prefer if the use of JSON was limited to the assignments_from_json class.

Reply:
JSON is now used/referenced only in `assignments_from_json` and the `get_user_specified_clinit_body` function in this file. The latter can be changed to allow other formats if/when we support them.